pkg/monitoring/metrics/virt-controller/vmistats_collector.go

```diff
@@ -61,7 +61,7 @@ var (
 			Name: "kubevirt_vmi_phase_count",
 			Help: "Sum of VMIs per phase and node. `phase` can be one of the following: [`Pending`, `Scheduling`, `Scheduled`, `Running`, `Succeeded`, `Failed`, `Unknown`].",
 		},
-		[]string{"node", "phase", "os", "workload", "flavor", "instance_type", "preference"},
+		[]string{"node", "phase", "os", "workload", "flavor", "instance_type", "preference", "guest_kernel_release", "guest_machine", "guest_name", "guest_version_id"},
```

Comment:
@assafad I think it would be better that the prefix would be `guest_os_`.

Reply:
Done